### DISCUSSION TITLE
.NET MAUI Embedding

### DISCUSSION THREAD

**Discussion by modula2019** [unoplatform/uno#18060] [Q&A]
hello all...please I need to use MAUI control like Picker in my uno platform app. I am using the .NET MAUI Embedding. I am able to display the picker perfectly in my UNO app. However, I want to connect the picker ItemsSource to a IListState property in the ViewModel. How can I set this property? I am using the MVUX data binding. Has anyone done this before? Thanks

Further information:
The viewmodel is in the uno project and I expect the maui embedded picker to use the ViewModel from the uno project
The repro is attached below

[UnoApp2.zip](https://github.com/user-attachments/files/16797979/UnoApp2.zip)

**Comment by MartinZikmund** [comment]
This is probably something @dr1rrb will be able to answer
